test/powershell/Modules/Microsoft.PowerShell.Core/RemoteImportModule.Tests.ps1

```diff
@@ -24,6 +24,8 @@ Describe "Remote import-module tests" -Tags 'Feature','RequireAdminOnWindows' {
         if ($IsWindows) {
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Dot-source operator is not allowed in modules on locked down systems that export functions with wildcards.Error occurred in Describe block
```powershell
Cannot bind argument to parameter 'Name' because it is null.
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\ConstrainedLanguageModules.Tests.ps1:260 char:27
```